### PR TITLE
Tune apiserver, kcm and scheduler flags for scale performance tests

### PR DESCRIPTION
**Pull request by hakuna-matatah**
- tune apiserver, kcm and scheduler flags for scale performance tests

### CODE REVIEW

**Comment by hakuna-matatah** [comment]
Looks like there seems to be github outage based on information from @hakman in slack, so tests are failing for that reason. 
We can resume the tests once github is back.

File: tests/e2e/scenarios/scalability/run-test.sh [hunk 1]
```diff
@@ -100,8 +100,21 @@ create_args+=("--set spec.etcdClusters[0].manager.listenMetricsURLs=http://local
...
 create_args+=("--set spec.kubeScheduler.authorizationAlwaysAllowPaths=/metrics")
+create_args+=("--set spec.kubeScheduler.kube-api-qps=500")
+create_args+=("--set spec.kubeScheduler.kube-api-burst=500")
+create_args+=("--set spec.kubeScheduler.profiling")
+create_args+=("--set spec.kubeScheduler.contention-profiling")
 create_args+=("--set spec.kubeControllerManager.endpointUpdatesBatchPeriod=500ms")
 create_args+=("--set spec.kubeControllerManager.endpointSliceUpdatesBatchPeriod=500ms")
+create_args+=("--set spec.kubeControllerManager.kube-api-qps=100")
+create_args+=("--set spec.kubeControllerManager.kube-api-burst=100")
+create_args+=("--set spec.kubeControllerManager.profiling")
```
Comment (hakman) [inline tests/e2e/scenarios/scalability/run-test.sh:111]:
```suggestion
create_args+=("--set spec.kubeControllerManager.kubeAPIQPS=100")
create_args+=("--set spec.kubeControllerManager.kubeAPIBurst=100")
create_args+=("--set spec.kubeControllerManager.enableProfiling")
```

File: tests/e2e/scenarios/scalability/run-test.sh [hunk 2]
```diff
@@ -100,8 +100,21 @@ create_args+=("--set spec.etcdClusters[0].manager.listenMetricsURLs=http://local
...
+create_args+=("--set spec.kubeControllerManager.contention-profiling")
+# inflight requests are bit higher than what currently upstream uses for GCE scale tests
+create_args+=("--set spec.kubeApiServer.max-requests-inflight=800")
+create_args+=("--set spec.kubeApiServer.max-mutating-requests-inflight=400")
+create_args+=("--set spec.kubeApiServer.profiling")
```
Comment (hakman) [inline tests/e2e/scenarios/scalability/run-test.sh:116]:
```suggestion
create_args+=("--set spec.kubeApiServer.maxRequestsInflight=800")
create_args+=("--set spec.kubeApiServer.maxMutatingRequestsInflight=400")
create_args+=("--set spec.kubeApiServer.enableProfiling")
```

**Comment by hakuna-matatah** [comment]
Ah! these changes are added in this [PR](https://github.com/kubernetes/kops/pull/15877/files) later,  thought we support all the flags as it is from k8s components today. Thanks for adding those quickly.

File: tests/e2e/scenarios/scalability/run-test.sh [hunk 3]
```diff
@@ -100,8 +100,21 @@ create_args+=("--set spec.etcdClusters[0].manager.listenMetricsURLs=http://local
...
+create_args+=("--set spec.kubeScheduler.kubeAPIQPS=500")
+create_args+=("--set spec.kubeScheduler.kubeAPIBurst=500")
+create_args+=("--set spec.kubeScheduler.enableProfiling")
+create_args+=("--set spec.kubeScheduler.enableContentionProfiling")
 create_args+=("--set spec.kubeControllerManager.endpointUpdatesBatchPeriod=500ms")
 create_args+=("--set spec.kubeControllerManager.endpointSliceUpdatesBatchPeriod=500ms")
+create_args+=("--set spec.kubeControllerManager.kubeAPIQPS=100")
+create_args+=("--set spec.kubeControllerManager.kubeAPIBurst=100")
+create_args+=("--set spec.kubeControllerManager.enableProfiling")
+create_args+=("--set spec.kubeControllerManager.enableContentionProfiling")
```
Comment (hakman) [inline tests/e2e/scenarios/scalability/run-test.sh:112]:
```suggestion
create_args+=("--set spec.kubeControllerManager.enableProfiling=true")
create_args+=("--set spec.kubeControllerManager.enableContentionProfiling=true")
```

**Review by hakman — CHANGES_REQUESTED** [review]

**Review by hakman — APPROVED** [review]

**Pull request merged**